### PR TITLE
add usage hints to README.md (and a few minor changes)

### PR DESCRIPTION
**Pull request by yoid2000**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,6 +61,16 @@ assumes each row represents a different entity.
 
 `--precision-limit-depth-threshold <int>` - Tree depth threshold below which the `row-fraction` check is not applied.
 
+## Usage hints
```
Comment (cristianberneanu) [inline README.md]: I think "Usage guide" is more appropriate here.

File: README.md [hunk 2]
```diff
@@ -61,6 +61,16 @@ assumes each row represents a different entity.
...
+
+The fewer columns that are included with the `--columns` parameter, the more accurate the resulting synthetic data. If for instance one is only interested in column pairs, then it is better to generate a different synthetic dataset for each pair than to take the pairs from one complete synthetic dataset.
+
+If the synthetic dataset is for the purpose of building a private ML model, and the target column of the model is known, then setting `--clustering-maincolumn` to the target column will produce synthetic data that leads to a better ML model.
```
Comment (cristianberneanu) [inline README.md]: "a private ML model" Why only private?

File: README.md [hunk 3]
```diff
@@ -61,6 +61,16 @@ assumes each row represents a different entity.
 
 `--precision-limit-depth-threshold <int>` - Tree depth threshold below which the `row-fraction` check is not applied.
 
+## Usage hints
+
+The fewer columns that are included with the `--columns` parameter, the more accurate the resulting synthetic data. If for instance one is only interested in column pairs, then it is better to generate a different synthetic dataset for each pair than to take the pairs from one complete synthetic dataset.
+
+If the synthetic dataset is for the purpose of building a private ML model, and the target column of the model is known, then setting `--clustering-maincolumn` to the target column will produce synthetic data that leads to a better ML model. 
+
+If there is more than one row per protected entity (i.e. per individual), then the `--aidcolumns` columns must be set.
```
Comment (cristianberneanu) [inline README.md]: "the `--aidcolumns` columns" should be "the `--aidcolumns` parameter"

**Comment by yoid2000** [comment]
Thanks Cristian. Updated.

**Review by cristianberneanu — APPROVED** [review]

**Pull request merged**